common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -5099,9 +5099,15 @@ schematic = (function() {
 	}
 
 	ConnectionPoint.prototype.toString = function() {
-	    return '<ConnectionPoint ('+this.offset_x+','+this.offset_y+') '+this.parent.toString()+'>';
+	    return edx.StringUtils.interpolate('<ConnectionPoint ({offset_x},{offset_y}) {parent}>',
+	    {
+		offset_x: this.offset_x,
+		offset_y: this.offset_y,
+		parent: edx.HtmlUtils.HTML(this.parent.toString())
```

Comment:
1. Are you able to test this?
2. Nit: Spacing is off in the PR.  Tabs?  Other?
3. A safer way to handle this would be:
```
parent: edx.HtmlUtils.ensureHtml(this.parent.toString())
```
See http://ui-toolkit.edx.org/utilities/html-utils/.
Note: I think these `toString()` functions are all now returning an `HtmlSnippet`.  In most cases, this should work fine where you want a String, but there are certain cases where it doesn't.  You should just ensure that this all still works as expected.  If it doesn't, there could be some places where you would need to use `componenet.toString().toString()`. :)